File: Auto-generated-tasks/Difficulty-2/Task-9/Solution.pas
Var
	Kol := 6, i1: integer;
	X: Array[0..Kol] of integer;
Begin
	for i1 := 0 to Kol do begin
		if (i1 < 2) then
			X[i1] := 1;
		else
			X[i1] := X[i1 - 2] + X[i1 - 1];
		WriteLn("X[", i1, "]=", X[i1]);
	end;
End.

File: Auto-generated-tasks/Difficulty-2/Task-9/Solution.f90
integer:: Kol = 6, i1
integer(len=Kol):: X
do i1 = 0, Kol
	if (i1 < 2)
		X(i1) = 1
	else
		X(i1) = X(i1 - 2) + X(i1 - 1)
	print *,"X[", i1, "]=", X(i1)
end do